Assets/_BForBoss/_Core/Scripts/Managers/GunRangeWorldManager.cs

```diff
@@ -4,13 +4,14 @@
 using Perigon.Weapons;
 using Sirenix.OdinInspector;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace BForBoss
 {
     public class GunRangeWorldManager : BaseWorldManager
     {
         [Title("Component")] 
-        [SerializeField] private WeaponsManager _weaponsManager = null;
+        [SerializeField] private WeaponAnimationController weaponAnimationController = null;
```

Comment:
I'll approve, this should be `_weaponAnimationController`. 
same with the other world manager

Reply:
Ah thats the rider rafactor screwing up here haha, I'll fix that and then merge